xrpl/utils/txn_parser/utils/types.py

```diff
@@ -25,17 +25,17 @@ class Balance(OptionalIssuer):
     """The amount of the currency."""
 
 
-class BalanceChange(TypedDict):
-    """A single balance change."""
+class ComputedBalance(TypedDict):
```

Comment:
Maybe a better name for this would be `AccountBalance`? 
(Computed doesn't seem to add much meaning)